docs/developers/tensorflow_coverage/tflite_base_converage.md

```diff
@@ -0,0 +1,165 @@
+# TensorFlow Lite Base Operation Coverage
+
+Tests of the TFLite's [builtin](https://www.tensorflow.org/mlir/tfl_ops)
+operation coverage.
+
+This focuses on lowering to `tosa` operations and may not have support for
+all IREE backends.
+
+*Last Updated: 2022/5/25
```

Comment:
How was this table generated?

Reply:
Hand entered. We don't have solid end-to-end comparison tests with tflite as tflite does not have op-by-op conformance tests.